### DISCUSSION TITLE
How do I save a preset made inside CTRLR au plugin?

### DISCUSSION THREAD

**Discussion by Sieber03** [RomanKubiak/ctrlr#338] [General]
How do I save a preset made inside CTRLR au plugin?

**Comment by Ted-juh** [comment]
Store it on the synth itself. Save it to a file? Some more context, please.

**Reply by Sieber03** [reply]
Thanks for your answer, I think I got a solution in this thread👌

**Comment by dobo365** [comment]
Indeed, would be easier with some context ;-)

Anyway, this is how I describe it in the manuals I'm providing with my panels (check any PDF on www.sunnysynths.com):

**Saving a patch as a Logic preset**
You can save the last patch saved in the panel as a plugin preset in Logic by selecting Save As in the pulldown menu of the preset area at the top of the plugin window. An .aupreset file will be created.
To replace a preset, just select another previously saved preset at the top left of the plugin window. All buttons will be positioned according to the newly loaded presets.

**Saving a patch as a Logic channel strip preset**
You can save the last patch saved in the panel as a channel strip preset in Logic by clicking on the Setting button at the top of the channel strip in the mixer and selecting Save Channel Strip Setting as…. Note that this is different than saving a plugin preset.
Creating a new track from a Logic channel strip setting
This is not possible directly but well in two steps. First, create a new Software Instrument track then click on the Setting button at the top of the channel strip in the mixer and select a previously saved channel strip setting.

**Reply by Sieber03** [reply]
Wauw amazing, I didn't think about this solution will try it asap 🥳🤘

**Reply by Sieber03** [reply]
Thanks for your answers 👌

**Reply by Ted-juh** [reply]
Please be so kind to mark Goodweather's answer as "the" answer.

**Reply by Sieber03** [reply]
> Please be so kind to mark Goodweather's answer as "the" answer.

How do I do that?

**Reply by damiensellier** [reply]
@Sieber03 click the "tick button" on the top right side of the good answer to your question.

![Capture d’écran, le 2021-08-16 à 15 25 46](https://user-images.githubusercontent.com/55164385/129618899-1a04a274-368b-40c2-b90f-03c86034036b.png)

**Comment by Sieber03** [comment]
![Screenshot 2021-08-16 at 21 34 47](https://user-images.githubusercontent.com/88163053/129619823-401c89ce-6625-46e5-bd47-0623b00f1bf9.png)
Anyone know how to fix this/these errors?

**Comment by dnaldoog** [comment]
You are trying to access a lua variable called **gDWBitMap**, but it doesn't exist. Either it has not been initialised at the time it was called or it has been declared `local` within the scope of an iterator or function and you are trying to access it outside that scope. or there is a typo (check case). I just spent an hour trying to work out why a call to `Amp 01` in a lua table wasn't working. The modulator name was `AMP 01`!

e.g.
<pre>
for i=0,10 do
local x=i
console (""..x)
end
console(""..x) -- will return error or nil
</pre>
